conf/arm.config

```diff
@@ -0,0 +1,262 @@
+
+    withName: TRIMGALORE {
+        container = { workflow.containerEngine == 'singularity' && !task.ext.singularity_pull_docker_container ? 'https://community-cr-prod.seqera.io/docker/registry/v2/blobs/sha256/10/10e7b91ef06bdf5790120809afc8cf36aac0261b618d810c952d5afd60dd5ed2/data' : 'community.wave.seqera.io/library/cutadapt_trim-galore:2aa2fc797328573b' }
+
+    }
+
+    //// No aarch64 builds in bioconda / conda-forge channels
+
+    //// a) have seqera aarch64 conda builds
+
+    withName: '.*:ALIGN_STAR:STAR_ALIGN_IGENOMES' {
+        //conda = 'seqera::star=2.6.1d bioconda::samtools=1.10 conda-forge::gawk=5.1.0'
+
+        conda = 'seqera::star=2.6.1d bioconda::samtools=1.21 conda-forge::gawk=5.1.0'
+        container = { workflow.containerEngine == 'singularity' && !task.ext.singularity_pull_docker_container ? 'https://community-cr-prod.seqera.io/docker/registry/v2/blobs/sha256/18/18b787cb710e54b55e04d8df4f5d363b3581c0f6008a908d7ab7de03275dfedb/data' : 'community.wave.seqera.io/library/star_samtools_gawk:c0246261b9d809e1' }
+    }
+
+    withName: 'STAR_GENOMEGENERATE_IGENOMES' {
+        //conda = 'bioconda::star=2.6.1d bioconda::samtools=1.10 conda-forge::gawk=5.1.0'
+
+        conda = 'seqera::star=2.6.1d bioconda::samtools=1.21 conda-forge::gawk=5.1.0'
+        container = { workflow.containerEngine == 'singularity' && !task.ext.singularity_pull_docker_container ? 'https://community-cr-prod.seqera.io/docker/registry/v2/blobs/sha256/18/18b787cb710e54b55e04d8df4f5d363b3581c0f6008a908d7ab7de03275dfedb/data' : 'community.wave.seqera.io/library/star_samtools_gawk:c0246261b9d809e1' }
+    }
+
+    withName: 'RSEM_PREPAREREFERENCE' {
+        //conda = 'bioconda::rsem=1.3.3 bioconda::star=2.7.10a'
+
+        conda = 'seqera::rsem=1.3.3 seqera::star=2.7.10a'
+        container = { workflow.containerEngine == 'singularity' && !task.ext.singularity_pull_docker_container ? 'https://community-cr-prod.seqera.io/docker/registry/v2/blobs/sha256/eb/eb5a2993bc13825b1b2443f740248c56ce0730a29553683392ce6388b7a2afe3/data' : 'community.wave.seqera.io/library/rsem_star:0dc50db916f3de8f' }
+    }
+
+    withName: 'RSEM_CALCULATEEXPRESSION' {
+        //conda = 'bioconda::rsem=1.3.3 bioconda::star=2.7.10a'
+
+        conda = 'seqera::rsem=1.3.3 seqera::star=2.7.10a'
+        container = { workflow.containerEngine == 'singularity' && !task.ext.singularity_pull_docker_container ? 'https://community-cr-prod.seqera.io/docker/registry/v2/blobs/sha256/eb/eb5a2993bc13825b1b2443f740248c56ce0730a29553683392ce6388b7a2afe3/data' : 'community.wave.seqera.io/library/rsem_star:0dc50db916f3de8f' }
+    }
+
+    //// b) need aarch64 conda builds
+
+    //withName: 'HISAT2_ALIGN' {
+    //    conda = 'bioconda::hisat2 bioconda::samtools'
+    //    //conda = 'bioconda::hisat2=2.2.1 bioconda::samtools=1.16.1'
+    //}
+
+    //withName: 'HISAT2_BUILD|HISAT2_EXTRACTSPLICESITES' {
+    //    conda = 'bioconda::hisat2'
+    //    //conda = 'bioconda::hisat2=2.2.1'
+    //}
+
```

Comment:
Do we need to add Kraken as an exclusion here too?

Reply:
Create individual issues for tools that aren't supported natively on Conda (using the `seqera` channel) and those that aren't supported at all.

Reply:
Done!